ray-operator/config/samples/pytorch-text-classifier/ray-job.pytorch-distributed-training.yaml

```diff
@@ -33,6 +33,7 @@ spec:
                   name: client
               resources:
                 limits:
+                  cpu: "2"
```

Comment:
This is mostly from my own experience, but others have shared similiar experience (e.g. https://home.robusta.dev/blog/stop-using-cpu-limits)

Reply:
Users can choose whether or not to use the CPU limit. In KubeRay, our goal is to ensure that the YAML manifests can be deployed across different Kubernetes clusters as smoothly as possible. Without a CPU limit, the Pod cannot be created in a namespace that has a ResourceQuota. Hence, KubeRay prefers to add resource limits to avoid this kind of issues.